tests/test_evaluators.py

```diff
@@ -472,5 +472,6 @@ def test_evaluation_structure(self):
             model=self.model,
             mapped_triples=self.triples_factory.mapped_triples,
             batch_size=1,
+            use_tqdm=False,
```

Comment:
maybe we should test using tqdm in some tests (otherwise we would not notice if we introduce a bug with tqdm somewhere).

Reply:
No ideas on how to do that come to mind. For now it seems like everything is working.

This change in the evaluator is just to reduce logging in the tests